sdk/search/Azure.Search.Documents/src/Batching/Publisher.cs

```diff
@@ -289,7 +289,7 @@ protected virtual async Task OnDocumentsAddedAsync(IEnumerable<T> documents, Can
                 _pending.Enqueue(action);
             }
 
-            AzureSearchDocumentsEventSource.Instance.PendingQueueResized(_pending.Count);
+            AzureSearchDocumentsEventSource.Instance.PendingQueueResized($"{GetType().Name}<{typeof(T).Name}>", _pending.Count);
```

Comment:
Is this dumping out Publisher as the name?  We shouldn't do that.  It's an internal implementation detail and not meaningful to any customer.  All of these should be `$"{nameof(SearchIndexingBufferedSender)}<{typeof(T).Name}>"`.

Reply:
Might also be useful to pull all this into a single helper instead of duplicating the logic in case we want to change it again before GA.